Gemfile

```diff
@@ -5,10 +5,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.4.0'
 
-gem 'carrierwave', '~> 0.11.2'
-gem 'jquery-rails', '~> 4.3', '>= 4.3.3'
 gem 'bootstrap', '~> 4.3.1'
+gem 'carrierwave', '~> 0.11.2'
 gem 'devise'
```

Comment:
There should be version numbers for devise , pundit, and rubocop.  Maybe we could fix this quickly?